LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -206,18 +197,29 @@ class LockboxXCUITests: BaseTestCase {
         searchTextField.typeText("a")
         // There should be two matches
         let twoMatches = app.tables.cells.count
-        XCTAssertEqual(twoMatches, 2)
-
+        if  UIDevice.current.userInterfaceIdiom == .pad {
```

Comment:
Could you try to do something similar to the [iPad method](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/BaseTestCase.swift#L112) tests will look nicer :)